templates/github/.github/workflows/scripts/script.sh.j2

```diff
@@ -56,27 +56,36 @@ if [[ "$TEST" == "plugin-from-pypi" ]]; then
   git checkout ${COMPONENT_VERSION} -- {{ plugin_name }}/tests/
 fi
 
+cat unittest_requirements.txt | cmd_stdin_prefix bash -c "cat > /tmp/unittest_requirements.txt"
+cat functest_requirements.txt | cmd_stdin_prefix bash -c "cat > /tmp/functest_requirements.txt"
+cmd_prefix pip3 install -r /tmp/unittest_requirements.txt
```

Comment:
If I'm not mistaken, cmd_prefix just runs `docker exec`. Would it make sense to move these steps where you're just installing extra requirements in the container to the `Containerfile`?

Reply:
Yes, that would make sense. I'll make that change.

Reply:
I tried implementing this and it's not going to work without doing a lot of refactoring to how we build the container. The current Containerfile template doesn't care if the plugin is being installed from source (PR time) or from a prebuilt wheel (for release pipeline). When the wheel is being used, the functest_requirements.txt and unittest_requirements.txt are not available during the built.